import-script/policies.json

```diff
@@ -15,6 +15,14 @@
     "role": "collection-author",
     "conditions": []
   },
+  {
```

Comment:
Very minor request, but would it be ok if we moved this to the end of the list after the other main default policy as there will be a lot of app specific policies in future and I wouldn't want the collection policy below to get buried in them.